config/global.yml

```diff
@@ -0,0 +1,16 @@
+exclude_titles:
+  - '[DO NOT MERGE]'
+  - 'DO NOT MERGE'
+  - "DON'T MERGE"
+  - 'DONT MERGE'
+  - 'WORK IN PROGRESS'
+  - 'WIP'
```

Comment:
If you're matching exact strings, you might also want to add `'[WIP]'` as well.  Sorry, I should've noted this before!